Library/Formula/bacula-fd.rb

```diff
@@ -13,6 +13,29 @@ class BaculaFd < Formula
   depends_on "readline"
   depends_on "openssl"
 
+  def plist; <<-EOS.undent
```

Comment:
You'll probably want a line like `plist_options :manual => "bacula-fd"` as well, depending on whether it must be root or not.